sdks/python/apache_beam/ml/gcp/recommendations_ai_test_it.py

```diff
@@ -40,6 +40,16 @@
 
 GCP_TEST_PROJECT = 'apache-beam-testing'
 
+CATALOG_ITEM = {
+    "id": str(int(random.randrange(100000))),
```

Comment:
Do we want to choose a different id than this? if we can do a string, could we have something like a date or descriptive ID prefix? Maybe `"aitest-%s-%d" % (datetime.now(). strftime("%Y%m%d-%H%M"), int(random.randint(1,10000)))` or something.

I also think that moving this to be global makes it harder to have multiple TestCases that create catalog items, but I'm not sure if we'll really be doing more tests. Would it make sense to have cleanup clean multiple items and not worry about the ID? like list + for each delete

Reply:
I'm ok with merging as is though since it's blocking the postcommit right now.